pax-std/pax-std-primitives/src/path.rs

```diff
@@ -36,33 +35,9 @@ impl InstanceNode for PathInstance {
 
     fn update(
```

Comment:
we can remove update entirely since there is a no-op default implementation instance node